app/app.go

```diff
@@ -718,12 +718,12 @@ func (app *App) RegisterUpgradeHandlers(cfg module.Configurator) {
 		minCommissionRate := sdk.NewDecWithPrec(5, 2)
 		for _, v := range validators {
 			if v.Commission.Rate.LT(minCommissionRate) {
-				comm, err := app.StakingKeeper.UpdateValidatorCommission(
-					ctx, v, minCommissionRate)
-				if err != nil {
-					panic(err)
+				if v.Commission.MaxRate.LT(minCommissionRate) {
+					v.Commission.MaxRate = minCommissionRate
```

Comment:
here you change the MaxRate, but you have to reset the value after have been able to change the commission rate

Reply:
I don't think it's a problem, because here he's checking if the max rate is _less than_ the min commission.

Reply:
max-commission should not be less than the new commission rate.
just be consistent.

Reply:
But is not related to the Rate or MinRate, but MaxRate.

Reply:
Oh yes ofc giansalex. Sorry!